src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests_LifetimeAnnotation.cs

```diff
@@ -223,7 +223,7 @@ static void Main()
             var method = comp.GetMember<MethodSymbol>("A.F4");
             Assert.Equal("void A.F4(ref scoped R r)", method.ToDisplayString(SymbolDisplayFormat.TestFormat.WithCompilerInternalOptions(SymbolDisplayCompilerInternalOptions.IncludeScoped)));
             var parameter = method.Parameters[0];
-            Assert.Equal(DeclarationScope.ValueScoped, parameter.Scope);
+            Assert.Equal(DeclarationScope.ValueScoped, parameter.DeclaredScope);
```

Comment:
Does the file need to be renamed?

Reply:
Yes, this test file should be renamed as part of [https://github.com/dotnet/roslyn/issues/62838](https://github.com/dotnet/roslyn/issues/62838).